docs/website/11ty/resolve-path.transform.js

```diff
@@ -26,11 +27,37 @@ class PathResolver {
     return window.document.documentElement.innerHTML;
   }
 
+  static isValidUrl(urlString) {
+		let url;
+
+		try { 
+	    url = new URL(urlString); 
+	  }
+	  catch(e) { 
+	    return false; 
+	  }
+
+	  return url.protocol === "http:" || url.protocol === "https:";
+	}
+
   static resolveLinks(dom, filePath) {
-    dom.querySelectorAll('a[href^="."]').forEach((link) => {
-      const resolved = PathResolver.resolveLink(link.href, filePath);
-      if (resolved !== link.href) console.info(blue(`Rewrite link "${link.href}" to "${resolved}"`));
-      link.href = resolved;
+    const domain = new URL(url).hostname;
+
+    dom.querySelectorAll('a[href]').forEach((link) => {
+      if (link.href.startsWith('.')) {
+        const resolved = PathResolver.resolveLink(link.href, filePath);
+
+        if (resolved !== link.href) console.info(blue(`Rewrite link "${link.href}" to "${resolved}"`));
+        link.href = resolved;
+      }
+      else if(PathResolver.isValidUrl(link.href)){
+        const linkDomain = new URL(link.href).hostname;
+      
+        if(domain !== linkDomain) {
+          link.target = "_blank";
+          link.rel = "noopener norefferer";
+        }
+      }
```

Comment:
mb can we do without this `isValidUrl` function?

```suggestion
      const linkDomain = new URL(link.href).hostname;
      
      if (linkDomain && domain !== linkDomain) {
        link.target = "_blank";
        link.rel = "noopener norefferer";
      }
```

Reply:
@abarmina If url is not valid, URL constructor throws TypeError. So we need to catch it, and it can be done in the loop. 
But I think it is better to keep this validation in separate function.